production/helm/loki/templates/NOTES.txt

```diff
@@ -34,3 +46,139 @@ Installed components:
 * query frontend
 {{- end }}
 {{- end }}
+
+
+{{- if eq (include "loki.deployment.isSingleBinary" .) "true" }}
```

Comment:
https://github.com/grafana/loki/blob/55b94903415c1bbe3a6cbd7d3a3a5f2d47f4f8a1/production/helm/loki/values.yaml#L30
Should this be `deploymentMode` ?

```suggestion
{{- if eq (include "loki.deploymentMode.isSingleBinary" .) "true" }}
```

Reply:
No this is correct. It's a template. I also tested this with `--dry-run` and all possible variations.